umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/iso19115_2.clj

```diff
@@ -361,6 +361,7 @@
          [:gmd:processingLevel
           [:gmd:MD_Identifier
            [:gmd:code (char-string (-> c :ProcessingLevel :Id))]
+           [:gmd:codeSpace (char-string "gov.nasa.esdis.umm.proessinglevelid")]
```

Comment:
Thanks, I copy and pasted from the spreadsheet and it was spelled wrong there too.  I have updated my spreadsheet and I put the string in a def.